app/src/main/java/com/github/geohunt/app/utility/DatabaseUtils.kt

```diff
@@ -21,6 +21,8 @@ inline fun <reified T> DataSnapshot.toMap(): Map<String, T>
     return map
 }
 
+inline fun <reified T> DataSnapshot.convertTo() : T? = this.getValue<T>()
```

Comment:
It would be worth checking in this function if the provided value is null. Also, a documentation would be appreciated. The final code could look like this one:

``/**
 * Converts the given [DataSnapshot] from Firebase to an instance of [T].
 *
 * @return The instance of [T], or null if the value is null or cannot be cast to type [T].
 */
inline fun <reified T> DataSnapshot.convertTo(): T? {
    val value = this.getValue<T>()
    return if (value is T) value else null
}``